tasks/bump.js

```diff
@@ -56,6 +56,16 @@ module.exports = function(grunt) {
         exactVersionToSet = false;
     }
 
+
+    //Use: --commitmsg="add me to commit message"
+    var appendToCommitMessage = grunt.option('commitmsg');
```

Comment:
if you are going to have a flag like `commitmsg` it should change the commit message, not append to it.